docs/app/pages/components/components-sections/tabs/tabs/tabs.component.html

```diff
@@ -62,6 +61,11 @@ <h4>{{ tab.title }}</h4>
     </tr>
 </uxd-api-properties>
 
+<uxd-api-properties tableTitle="Methods">
+    <tr uxd-api-property name="selectTab" args="tab: number | TabComponent">
+        Programmatically select a tab. Method accepts a tab index or a tab instance as an arguement. The index is zero based i.e. the first tab will be 0.
```

Comment:
Small typo, arguement -> argument.

Reply:
Spelling was never a strength of mine haha